docs/src/pages/reference/configuration/output.md

```diff
@@ -2082,3 +2082,20 @@ module.exports = {
   },
 };
 ```
+
+### propertySortOrder
+
+Type: `Alphabetical` | `Specification`
+
+Default Value: `Alphabetical`
```

Comment:
It would make more sense to have specification by default no?

Reply:
I mean, it would, yes, but then why were we sorting alphabetically from the beginning? You get the specification sort by not doing any sorting